storybook/src/example-pages/toepassingen/vergunningscheck/landingspagina.ts

```diff
@@ -127,10 +127,10 @@ examplePageFactory(
                     `,
                   }),
                 })}
-              </div>
-            </div>
-            <div class="row dso-equal-heights">
-              <div class="col-md-3 col-sm-6">
+              </li>
+            </ol>
+            <ol class="row dso-equal-heights" start="5">
```

Comment:
Idem. Ik heb 't net getest in de browser, visueel ziet het er nog best wel spiffy uit. Accessibility wise begrijp ik ook niet waarom we hier 2 lijsten van maken.

En hoe gaat de AT om met `display: flex` op li? De accessibility inspector van Chrome lijkt het goed als list item te tonen

Reply:
Heb van de 2 ol's 1 ol gemaakt.